merlin-server/constraints-dsl-compiler/src/libs/constraints-ast.ts

```diff
@@ -68,6 +70,13 @@ export interface WindowsExpressionInvert {
   expression: WindowsExpression;
 }
 
+export interface WindowsExpressionShiftBy {
+  kind: NodeKind.WindowsExpressionShiftBy,
+  windowExpression: WindowsExpression,
+  fromStart: number,
```

Comment:
:large_orange_diamond: There is a `type Duration = number` on line 482. Should we be using that time here?

Reply:
Seconded: I added that line, it is currently not used. I only added it because that same line existed in `scheduling-ast.ts` too as a placeholder. When we replace `Duration` with a more appropriate type, we'll have to change this to `Duration` anyway.

Reply:
It is in the constraints edsl, not the ast. But I have added it to the AST and used it in this node.